packages/api/src/controllers/helpers.ts

```diff
@@ -299,12 +299,47 @@ export async function sendgridEmail({
 export async function sendgridEmailPaymentFailed({
   email,
   sendgridApiKey,
-  userId,
+  user,
   invoiceId,
+  invoiceUrl,
+  templateId,
 }) {
+  // If user.lastPaymentFailed is newer than 3 days, don't send email
+  if (
+    user.lastPaymentFailed &&
+    user.lastPaymentFailed > Date.now() - 3 * 24 * 60 * 60 * 1000
+  ) {
+    return false;
+  }
+
   const [supportName, supportEmail] = email;
+
+  let subject: string;
+  let text: string;
+
+  if (!invoiceId) {
+    subject = `Free tier user reached usage limit`;
+    text = `User ${user.email} reached usage limit`;
+  } else {
+    subject = `Payment failed for invoice ${invoiceId} for user ${email}`;
+    text = `User ${user.email} failed to pay invoice ${invoiceId}`;
+  }
+
   const msg = {
-    text: `User ${userId} failed to pay invoice ${invoiceId}`,
+    personalizations: [
+      {
+        to: [{ email }],
+        bcc: undefined,
```

Comment:
Should these emails go to an internal address we can track?

Reply:
the `email` receiver it's already an internal address, do you think we should send it to a secondary email?

Reply:
That's fine then, I thought the email was going to the user